### PR TITLE
Removed jquery alias, replaced it with externals property

### PR DESCRIPTION
**Pull request by ptsavdar**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Fixed tickets | no

Removed jquery alias in webpack config, replaced it with externals property. Alias was causing problems with third party plugins such as fullCalendar.

### CODE REVIEW

**Comment by ptsavdar** [comment]
You can test that the environment with new configuration here http://pr-1459-1476968908.travis-build.claroline.net/claroline/

File: main/core/Resources/scripts/lib/webpack.js [hunk 1]
```diff
@@ -73,10 +73,13 @@ function configure (rootDir, packages, isWatchMode) {
...
     entry: entries,
     output: output,
+    externals: {
+      // require("jquery") is external and available on the global var jQuery
+      'jquery': 'jQuery'
+    },
     resolve: {
       root: root,
       alias: {
-          jquery: __dirname + '/../../modules/jquery',
```
Comment (stefk) [inline main/core/Resources/scripts/lib/webpack.js]: Can you remove the corresponding as well?

Comment (ptsavdar) [inline main/core/Resources/scripts/lib/webpack.js]: Ok, done ;)

**Comment by stefk** [comment]
LGTM

**Comment by ptsavdar** [comment]
LGTM

**Pull request merged**